### PR TITLE
Add arguments required for OpenAI connection

### PR DESCRIPTION
**Pull request by wdfox**
## Purpose

* Commit [c8b8486](https://github.com/Azure-Samples/azure-search-openai-demo/commit/c8b8486d68527026b0660a3a75cc4e5e376793c1) removed two command line arguments to the `prepdocs.sh` and `prepdocs.ps1` scripts which break the document loading process for users of the external OpenAI connection. This has been addressed in the open issue #676 
* I added the two arguments (`openaihost` and `openaikey`) back into the file, fixing the problem.
* Based on the logic of `prepdocs.py` (specifically lines 736-752), I don't believe the change will impact users of the Azure OpenAI service. However, I don't have access, so I cannot test this. 

## Does this introduce a breaking change?

```
[ ] Yes
[X ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[X ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

Run `azd up` and specifically watch the `prepdocs` process. There should be no errors for users of the external OpenAI connection.

## What to Check
Verify that the following are valid
* I only have the ability to run in a Linux environment with 3rd party OpenAI connection. 
* It would help for a maintainer to validate that the change (1) doesn't affect users of the Azure OpenAI service, and (2) the changes to `prepdocs.ps1` solve the issue for Windows users.

### CODE REVIEW

**Comment by wdfox** [comment]
@microsoft-github-policy-service agree

File: scripts/prepdocs.ps1 [hunk 1]
```diff
@@ -33,5 +33,6 @@ $argumentList = "./scripts/prepdocs.py `"$cwd/data/*`" $adlsGen2StorageAccountAr
 "--openaiservice `"$env:AZURE_OPENAI_SERVICE`" --openaikey `"$env:OPENAI_API_KEY`" " + `
 "--openaiorg `"$env:OPENAI_ORGANIZATION`" --openaideployment `"$env:AZURE_OPENAI_EMB_DEPLOYMENT`" " + `
 "--openaimodelname `"$env:AZURE_OPENAI_EMB_MODEL_NAME`" --index $env:AZURE_SEARCH_INDEX " + `
-"--formrecognizerservice $env:AZURE_FORMRECOGNIZER_SERVICE --tenantid $env:AZURE_TENANT_ID -v"
+"--formrecognizerservice $env:AZURE_FORMRECOGNIZER_SERVICE --tenantid $env:AZURE_TENANT_ID" + `
+"--openaihost $env:OPENAI_HOST --openaikey $env:OPENAI_API_KEY -v"
```
Comment (pamelafox) [inline scripts/prepdocs.ps1:37]: I see host and key on lines 32-33 so I think they didnt get dropped from this one?

**Review by pamelafox — CHANGES_REQUESTED** [review]
Thanks so much, 1 comment

File: scripts/prepdocs.ps1 (hunk 1 above)
Comment (wdfox) [reply scripts/prepdocs.ps1]: My mistake. You’re spot on. Only the `prepdocs.sh` file needs to be changed. Just removed the unnecessary args from the Powershell script.

**Review by pamelafox — APPROVED** [review]

File: scripts/prepdocs.ps1 (hunk 1 above)
Comment (namratam30) [reply scripts/prepdocs.ps1]: I am confused between OpenAI Orgnization and openaihost. Could you please help me how to pass these details-
OPENAI_API_TYPE = "azure"
OPENAI_API_VERSION = "2023-07-01-preview"
OPENAI_API_KEY = "xxxxxxx"
OPENAI_API_BASE = " https://xxxxxx.openai.azure.com/"

Comment (pamelafox) [reply scripts/prepdocs.ps1]: Replied to your comment on other thread

**Pull request merged**